spring-kafka/src/test/java/org/springframework/kafka/support/micrometer/ObservationTests.java

```diff
@@ -276,6 +325,20 @@ KafkaTemplate<Integer, String> customTemplate(ProducerFactory<Integer, String> p
 			return template;
 		}
 
+		@Bean
+		KafkaTemplate<Integer, String> runtimeExceptionTemplate(ProducerFactory<Integer, String> pf) {
+			KafkaTemplate<Integer, String> template = new KafkaTemplate<>(pf);
+			template.setObservationEnabled(true);
+			return template;
+		}
+
+		@Bean
+		KafkaTemplate<Integer, String> errorTemplate(ProducerFactory<Integer, String> pf) {
+			KafkaTemplate<Integer, String> template = new KafkaTemplate<>(pf);
+			template.setObservationEnabled(true);
+			return template;
```

Comment:
Looks like both of these new `KafkaTemplate` beans are just exactly the same what is `template` bean.

Any particular reason why do we need these two?
Why that `template` is not enough for new tests?

Thanks

Reply:
Just for test tag `spring.kafka.template.name`, is doesn't matter.
Should I keep it or remove?